components/mdc/Datatable/DatatableCheckbox.svelte

```diff
@@ -9,7 +9,13 @@ onMount(() => dispatch('mounted'))
 const dispatch = createEventDispatcher()
 </script>
 
-<td class="mdc-data-table__cell mdc-data-table__cell--checkbox" on:click on:keypress on:keyup on:keydown>
+<td
+  class="mdc-data-table__cell mdc-data-table__cell--checkbox {$$props.class}"
```

Comment:
Is this the kind of thing that @forevermatt was suggesting?
```suggestion
  class="mdc-data-table__cell mdc-data-table__cell--checkbox {$$props.class || ''}"
```

Reply:
yes it is, I was going to do them all in a seperate PR.